### ISSUE TITLE
Incorrect parameter type message for Number parameters

### ISSUE THREAD

**Issue by reidca** [awslabs/cfn-python-lint#146]
Receiving the error:

```
E2530 Type for Parameter should be Integer, MinValue should be at least 128, and MaxValue equal or less than 1536 at Parameters/MemorySize/Type
cloudformation/ec2/management/templates/instance-scheduler.yaml:20:5
```

The block of cloudformation it is complaining about it:

```
MemorySize:
    Type: Number
    AllowedValues:
      - 128
      - 384
      - 512
      - 640
      - 768
      - 896
      - 1024
      - 1152
      - 1280
      - 1408
      - 1536
    Default: 128
    Description: Size of the Lambda function running the scheduler, increase size when
      processing large numbers of instances
```

There is no parameter type of Integer so I don't where this error is coming from?

**Comment by kddejong** [comment]
I'm almost done with this one.
